### PR TITLE
testing branch commit

### PR DESCRIPTION
**Pull request by serje-havandjian**
Testing example branch

### CODE REVIEW

**Comment by serje-havandjian** [comment]
Learning branches, pull requests @ryanhburbank

File: client/src/index.css [hunk 1]
```diff
@@ -8,7 +8,7 @@ body {
 
   background-image: url(https://clipart.world/wp-content/uploads/2021/04/Gavel-clipart-transparent.png);
   background-repeat: no-repeat;
-  background-color: rgba(68, 63, 63, 0.735)
+  background-color: rgba(100, 100, 100, 0.735)
```
Comment (ryanhburbank) [inline client/src/index.css:11]: let's use this color instead `(99, 99, 99, 0.735)`

File: client/src/components/Window.js [hunk 2]
```diff
@@ -137,7 +137,58 @@ function WindowApp({displayEditForm, setDisplayEditForm, setIndividualCase, laws
         e.target.reset()
     } 
 
+    // Function to render DISCOVERY SERVED DEADLINES (NOT YET WORKING!)
```
Comment (ryanhburbank) [inline client/src/components/Window.js:140]: if this is commented out, can we delete it?

**Review by ryanhburbank — CHANGES_REQUESTED** [review]
please see comments for changes

**Review by ryanhburbank — APPROVED** [review]
LGTM

**Pull request merged**